### DISCUSSION TITLE
How to generate `TOC` with special chars in .md file?

### DISCUSSION THREAD

**Discussion by astraw99** [community/community#27994] [Other Feature Feedback, Questions, & Ideas] [answered]
In my .md file, want to add `TOC` of all the titles and subtitles, like below:
```
TOC
- [1. Docker ENTRYPOINT vs CMD](#1-Docker-ENTRYPOINT-vs-CMD)
    - [1.1 Only ENTRYPOINT](#11-Only-ENTRYPOINT)
    - [1.2 Both ENTRYPOINT + CMD](#12-Both-ENTRYPOINT---CMD)
```

In the link part, the space char is replaced with '-', but how to replace the '+' char?
Tried with '---' to replace '+', but it didn't work. Anyone help?

**Comment by airtower-luna** [comment, marked as answer]
If you hover the mouse over the link symbol that appears if your mouse is over a heading, your browser should show you the direct link. Look at the anchor part of that to figure out the anchor you need to use, that works for any characters.

I mean the symbol on the left:

![grafik](https://user-images.githubusercontent.com/5814919/182307426-d0349b39-94d6-4ef9-9167-a2e1cdc13697.png)

**Reply by astraw99** [reply]
Got it, thanks 👍
